src/components/layout/MobileNavigation.vue

```diff
@@ -79,7 +79,7 @@ export default {
   computed: {
     ...mapState(['useSdkWallet']),
     isTipsRoute() {
-      return ['tips', 'tips-search'].includes(this.$route.name);
+      return ['tips', 'feed-search'].includes(this.$route.name);
```

Comment:
```suggestion
      return ['feed', 'feed-search'].includes(this.$route.name);
```
I think `tips` route also should be renamed